### PR TITLE
Ch.9 proofs

### PR DESCRIPTION
**Pull request by emilynagler**

### CODE REVIEW

File: 09-next.Rmd [hunk 1]
```diff
@@ -4,75 +4,78 @@ editor_options:
...
-This is more of a *framework* than a package, providing a powerful system for generating reports, web pages and even books (this book was written with the `bookdown` package, which builds on `rmarkdown`).
+An advantage of R is that it has many packages dedicated to the communication and publication of results, vital for policy impact. 
+Perhaps the most important important package with regards to the communication of results is rmarkdown. 
+This is more of a framework than a package, providing a powerful system for generating reports, web pages and even books (this book was written with the bookdown package, which builds on rmarkdown).
 
-Here is not the place to explain how to use RMarkdown.
-The framework is explained in detail in a free and open book [@xie_markdown_2018].
-To get started with the framework, however, you can try the following (code example showing the creation of an Rmd file):
+Here is not the place to explain how to use RMarkdown (Rmd). 
+The framework is explained in detail in a free and open book (Xie et al., 2018).
```
Comment (Robinlovelace) [inline 09-next.Rmd:43]: Heads-up @emilynagler the notation `[@xie_markdown_2018]` is deliberate: it automatically creates a citation.

File: 09-next.Rmd [hunk 2]
```diff
@@ -4,75 +4,78 @@ editor_options:
...
+If you, at any point, become frustrated due to tricky-to-fix error messages (something that all users of command-line software have experienced at some point!), you may want to skip to Section 9.4, written by people who have navigated R’s at first daunting learning curve to arrive safely at place where R saves more time than it consumes and takes your work ‘to the next level.’
 
 ## Automated reporting with RMarkdown
 
-An advantage of R is that it has many packages dedicated to the communication and publication of results, vital for policy impact.
-Perhaps the most important important package for communication of results is `rmarkdown`.
-This is more of a *framework* than a package, providing a powerful system for generating reports, web pages and even books (this book was written with the `bookdown` package, which builds on `rmarkdown`).
+An advantage of R is that it has many packages dedicated to the communication and publication of results, vital for policy impact. 
+Perhaps the most important important package with regards to the communication of results is rmarkdown.
```
Comment (Robinlovelace) [inline 09-next.Rmd:39]: I think packages are well-represented as `monotype` - that is what the backticks around 'rmarkdown' do.

File: 09-next.Rmd [hunk 3]
```diff
@@ -4,75 +4,78 @@ editor_options:
...
+An advantage of R is that it has many packages dedicated to the communication and publication of results, vital for policy impact. 
+Perhaps the most important important package with regards to the communication of results is rmarkdown. 
+This is more of a framework than a package, providing a powerful system for generating reports, web pages and even books (this book was written with the bookdown package, which builds on rmarkdown).
 
-Here is not the place to explain how to use RMarkdown.
-The framework is explained in detail in a free and open book [@xie_markdown_2018].
-To get started with the framework, however, you can try the following (code example showing the creation of an Rmd file):
+Here is not the place to explain how to use RMarkdown (Rmd). 
+The framework is explained in detail in a free and open book (Xie et al., 2018). 
+To get started with the framework, however, you can try the following code example, which shows the creation of an Rmd file:
```
Comment (Robinlovelace) [inline 09-next.Rmd:44]: :+1:

File: 09-next.Rmd [hunk 4]
```diff
@@ -4,75 +4,78 @@ editor_options:
...
 
 ```{r, eval=FALSE}
 file.edit("test-document.Rmd")
 ```
 
-Try adding some code chunks and text by following the guidance in the RMarkdown cheatsheet, which you can get from the Help > Cheatsheets menu in RStudio.
+Try adding some code chunks and text by following the guidance in the Rmd cheatsheet, which you can get from the Help > Cheatsheets menu in RStudio.
```
Comment (Robinlovelace) [inline 09-next.Rmd:50]: :+1:

File: 09-next.Rmd [hunk 5]
```diff
@@ -87,17 +90,15 @@ reprex::reprex({
...
 ```
 
-Note, you can turn any bit of code into a 'reprex' by selecting it and by running the 'Reprex selection' Addin in RStudio, as described on the [Tidyverse website](https://www.tidyverse.org/help/).
-
+Note, you can turn any bit of code into a 'reprex' by selecting it and by running the 'Reprex selection' addin in RStudio, as described on the [Tidyverse website](https://www.tidyverse.org/help/).
 
 ## Testimonials
 
-This final section provides insight into how R can be used for road safety research from a range of perspectives.
+This final section provides insight not only into how R can be used for road safety research from a range of perspectives, but also navigating R's at times steep learning curve.
```
Comment (Robinlovelace) [inline 09-next.Rmd:97]: :+1:

File: 09-next.Rmd [hunk 6]
```diff
@@ -138,39 +139,42 @@ Thinking of myself as an "Excel native", I would say R hasn't replaced Excel, bu
...
-The huge Shiny ecosystem has allowed us to produce interactive applications for sharing our tools and visualisations with stakeholders.
-There is also a wide range of open-source statistical and machine learning packages available which, when combined with the added capability to translate and use tools from python, has allowed us to innovate and take full advantage of what artificial intelligence has to offer.
-All of this, embedded in a supportive community of R users that is continually sharing its knowledge and helping spread these skills, means that anything is possible for users with any level of experience.
-The fact that this community includes a growing number of road safety analysts and transport-focused data scientists (thanks, in part, to this book and the previous *R for Road Safety* courses) means that there are plenty of like-minded R users all over the country that you can share ideas (and code) with.
+After attending my first R course, I immediately saw how useful the language could be to our team. 
+At Agilysis, we have integrated R into almost everything our analytics team does, pushing for our work to be as robust and reproducible as possible.
```
Comment (Robinlovelace) [inline 09-next.Rmd:143]: Did you mean to remove the link to the company @emilynagler ?

File: 09-next.Rmd [hunk 7]
```diff
@@ -182,3 +186,10 @@ Do you have a use case of reproducible research?
 Please get in touch on the [issue tracker](https://github.com/ITSLeeds/rrsrr/issues).
 
 # References
```
Comment (Robinlovelace) [inline 09-next.Rmd:188]: These are auto-generated so no need to re-add them.

**Review by Robinlovelace — COMMENTED** [review]
Hi @emilynagler, thanks for these. I have requested a few changes. Let me know if you'd like to respond and give them a bash. Alternatively I can just merge this, which also has some good changes (many thanks) and tidy up after.

File: 09-next.Rmd (hunk 1 above)
Comment (emilynagler) [reply 09-next.Rmd:43]: good to know

**Comment by emilynagler** [comment]
> Hi @emilynagler, thanks for these. I have requested a few changes. Let me know if you'd like to respond and give them a bash. Alternatively I can just merge this, which also has some good changes (many thanks) and tidy up after.

Hi @Robinlovelace I would say just merge them, sorry for the formatting errors here and there. I'm trying to be more careful on the next chapter. I think I copied and pasted chunks that were too big and didn't go through them carefully enough.

File: 09-next.Rmd (hunk 6 above)
Comment (emilynagler) [reply 09-next.Rmd:143]: no probably not, sorry! I must have copied that bit straight from the word doc.

File: 09-next.Rmd (hunk 2 above)
Comment (emilynagler) [reply 09-next.Rmd:39]: I'll keep that in mind in the next chapter!

**Comment by Robinlovelace** [comment]
Hi @emilynagler no worries, have merged, look forward to your comments on the next chapter.

**Pull request merged**